python/openai/openai_frontend/engine/utils/triton.py

```diff
@@ -40,26 +40,30 @@ def _create_vllm_inference_request(
 
     # NOTE: The exclude_none is important, as internals may not support
     # values of NoneType at this time.
-    sampling_parameters = request.model_dump(
+    sampling_parameters = request.model_dump_json(
         exclude=excludes,
         exclude_none=True,
     )
-    inputs["text_input"] = [prompt]
-    inputs["stream"] = [request.stream]
     exclude_input_in_output = True
     echo = getattr(request, "echo", None)
     if echo:
         exclude_input_in_output = not echo
-    inputs["exclude_input_in_output"] = [exclude_input_in_output]
-    return model.create_request(inputs=inputs, parameters=sampling_parameters)
+
+    inputs["text_input"] = [prompt]
+    inputs["stream"] = np.bool_([request.stream])
+    inputs["exclude_input_in_output"] = np.bool_([exclude_input_in_output])
+    # Pass sampling_parameters as serialized JSON string input to support List
+    # fields like 'stop' that aren't supported by TRITONSERVER_Parameters yet.
+    inputs["sampling_parameters"] = [sampling_parameters]
+    return model.create_request(inputs=inputs, parameters={})
```

Comment:
nit: can remove parameters, keeping empty can be a little confusing at first look

Reply:
done: https://github.com/triton-inference-server/server/pull/7682/commits/9f17b62ecec2ba6b2372d1f0252dcd7355075b86